libraries/radpdfprocessing/formats-and-conversion/pdf/pdfformatprovider/settings.md

```diff
@@ -107,6 +137,27 @@ __Example 2__ shows how you can create a __PdfExportSettings__ object and assign
 
 {{endregion}}
 
+__Example 3__ shows how you can create a __PdfExportSettings__ object with a different set of settings. The settings used specify an OwnerPassword and UserAccessPermissions.
```

Comment:
This example is specifically related to setting permissions. Move it just after the description for these permissions. Don't forget to update the numbering of the snippets